packages/react-examples/src/react-menu/MenuList/MenuList.stories.tsx

```diff
@@ -27,25 +25,45 @@ const Container: React.FC = props => {
 };
 
 export const MenuListExample = () => (
-  <FluentProvider theme={teamsLightTheme}>
-    <Container>
-      <MenuList>
-        <MenuItem>Item</MenuItem>
-        <MenuItem>Item</MenuItem>
-        <MenuItem>Item</MenuItem>
-      </MenuList>
-    </Container>
-  </FluentProvider>
+  <Container>
+    <MenuList>
+      <MenuItem>Item</MenuItem>
+      <MenuItem>Item</MenuItem>
+      <MenuItem>Item</MenuItem>
+    </MenuList>
+  </Container>
 );
 
 export const MenuListWithIconsExample = () => (
-  <FluentProvider theme={teamsLightTheme}>
+  <Container>
+    <MenuList>
+      <MenuItem icon={<CutIcon />}>Item</MenuItem>
+      <MenuItem icon={<PasteIcon />}>Item</MenuItem>
+      <MenuItem icon={<EditIcon />}>Item</MenuItem>
+    </MenuList>
+  </Container>
+);
+
+export const MenuListWithCheckboxes = () => {
+  const checkmark = <AcceptIcon />;
+  const [checkedValues, setCheckedValues] = React.useState<Record<string, string[]>>({});
+  const onChange = (name: string, items: string[]) => {
+    setCheckedValues(s => ({ ...s, [name]: items }));
+  };
+
+  return (
     <Container>
-      <MenuList>
-        <MenuItem icon={<CutIcon />}>Item</MenuItem>
-        <MenuItem icon={<PasteIcon />}>Item</MenuItem>
-        <MenuItem icon={<EditIcon />}>Item</MenuItem>
+      <MenuList checkedValues={checkedValues} onCheckedValuesChange={onChange}>
+        <MenuItemCheckbox name="checkbox" value="1" checkmark={checkmark}>
```

Comment:
what happens if you do not provide `checkmark`?

Reply:
I want to set a default but also don't want to depend on non-converged mdl2 icons

Reply:
workaround for story purposes is to manually set the slot here